client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -35,10 +42,14 @@ export function getEnhancedTasks(
 	goToStep?: NavigationControls[ 'goToStep' ],
 	flow?: string | null,
```

Comment:
[nit] It might be better to give it a default value and then we don't need to use `flow || null` all the time
```suggestion
	flow: string | null = '',
```